doc/display.xml

```diff
@@ -110,6 +111,10 @@ gap> Splash(DotDigraph(RandomDigraph(4)));
       vertices, with the arrowhead of each line pointing towards the range
       of the edge.<P/>
 
+      <C>DotColoredDigraph</C> differs from <C>DotDigraph</C> only in
+      that the values in given in the two lists are used to color the vertices and
+      edges of the graph when displayed.  <P/>
```

Comment:
I think you should add more details about what values the arguments can have. I.e. what are the possible "colors" of the vertices and edges? what if I want to specify vertex colors but not edge colors, or vice versa? the list of edge colors has to have the same "shape" as the list of `Outneighbours` right?